### DISCUSSION TITLE
Cost invoices - how is it working like

### DISCUSSION THREAD

**Discussion by waran70** [YetiForceCompany/YetiForceCRM#14259] [How to]
just touched new thing for me - cost invoices. It has no ui10 link to any other module... 
How it is working like?

**Comment by hahyes** [comment]
> just touched new thing for me - cost invoices. It has no ui10 link to any other module...
> How it is working like?

This is the same thing as proform invoices, there is no even transision from proform to selling invoice. You need to create mappings and relations for your own needs. Cost invoices could be related to single orders for example.
